content/influxdb/cloud/visualize-data/annotations.md

```diff
@@ -21,25 +21,24 @@ Add annotations to your dashboards to provide useful, contextual information abo
 Annotations may be useful to highlight operations or anomalies for your team to reference.
 {{% /note %}}
 
-{{< youtube 5NEplCesNAc >}}
+<!-- {{< youtube 5NEplCesNAc >}} --->
 
 ## Create an annotation
 
 1. In the navigation menu on the left, select **Boards** (**Dashboards**).
 
     {{< nav-icon "dashboards" >}}
 
-2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/).
-3. Click **Annotations**. The **Show Annotations** option is selected by default.
-4. Select the **Enable 1-Click Annotations** option, and then in any dashboard cell, click the point in time to add the annotation.
-  {{% note %}}
-**Tip:** To move the annotation outside of the selected graph time range, first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
-  {{% /note %}}
-5. On the **Add Annotation** page:
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/). The **Annotations** option is selected by default.
+3. In a dashboard cell, doing one of the following:
```

Comment:
Is `doing` the right form here? Should it be `do`? None of the other verbs are in present participle

Reply:
```suggestion
3. In a dashboard cell, do one of the following:
```